docs/source/tutorials.rst

```diff
@@ -5,17 +5,30 @@ Tutorials
 
 
 Here a set of examples on how to use different MyoSuite models and non-stationarities.
-Jupyter-Notebooks can be found `here <https://github.com/facebookresearch/myosuite/tree/main/docs/source/tutorials>`__
+It is highly recommended to read through the `OpenAI Gym API <https://gymnasium.farama.org/>`__ before you start.
 
 * :ref:`run_myosuite`
 * :ref:`run_visualize_index_movements`
 * :ref:`run_trained_policy`
-* :ref:`test_muscle_fatigue`
-* :ref:`test_sarcopenia`
-* :ref:`test_tendon_transfer`
-* :ref:`resume_training`
-* :ref:`load_deprl_baseline`
-* :ref:`load_MyoReflex_baseline`
+* :ref:`advanced_muscle_conditions`
+    * :ref:`test_muscle_fatigue`
+    * :ref:`test_sarcopenia`
+    * :ref:`test_tendon_transfer`
+    * :ref:`exoskeleton`
+* :ref:`use_reinforcement_learning`
+    * :ref:`resume_training`
+    * :ref:`load_deprl_baseline`
+    * :ref:`load_MyoReflex_baseline`
+* :ref:`customizing_tasks`
+
+.. _jupyter_notebook:
+
+Tutorials on Jupyter-Notebook
+========================================
+Please refer to our tutorials on Jupyter-Notebook `here <https://github.com/facebookresearch/myosuite/tree/main/docs/source/tutorials>`__
```

Comment:
@raku-slyu - add a short summary of what the tutorial covers

Reply:
include the phrase "key functionalities, such as model usage and examples of using RL"